### ISSUE TITLE
Remove the CSS rules for quotes

### ISSUE THREAD

**Issue by joyously** [Automattic/_s#1192]
These rules remove the default quotes set up by browsers. Most theme authors don't add anything to style the `q` tag.  I think these rules should be removed, so the browser defaults are used. At least the defaults handle nested quotes nicely. With these rules, no quotes at all!

```
blockquote:before,
blockquote:after,
q:before,
q:after {
	content: "";
}

blockquote,
q {
	quotes: "" "";
}

```

**Comment by grappler** [comment]
@joyously Do you have some screenshots how it looks now and how it would look like when the browser defaults are used?

**Comment by joyously** [comment]
These are the browser defaults for `q` tags. The browser does not have quote defaults for `blockquote` as far as I can see.
```
q::before {  content: open-quote; }
q::after  {  content: close-quote; }
```
**Here is the HTML for the example:**
`<p>My brother told me a joke that ended with, <q>And then she said <q>Put that <em>away</em>!</q> before I could do anything.</q> It was funny, but I don't remember it all.</p>`

**Here is what it looks like now (no changes):**
![nested-q-now](https://user-images.githubusercontent.com/7957680/29218732-58f43946-7e7b-11e7-97bd-c53eafab46ef.jpg)

**Here is what it should look like (browser defaults):**
![nested-q-rules-removed](https://user-images.githubusercontent.com/7957680/29218731-58f26454-7e7b-11e7-85b3-7ed62d1693d5.jpg)

**Comment by grappler** [comment]
After doing some research I agree. The removing of the quotes style was a remainder from [Eric Meyer's reset](http://meyerweb.com/eric/tools/css/reset/index.html). 6c47be98c133c911c1533648225e3b720ba260d4

[Normalize does not touch the quotes CSS](https://github.com/necolas/normalize.css/issues/493#issuecomment-168899569) so that the browsers can support the different quotation marks in the different languages.

I made a little test to show how it would look in the different languages without the styling. https://jsfiddle.net/9gtfyak1/1/

Some really interesting information on https://en.wikipedia.org/wiki/Quotation_mark

**Comment by davidakennedy** [comment]
Thanks for creating this issue @joyously and for the additional information @grappler.

I think this is a good move.

**Comment by grappler** [comment]
Closing as it as been fixed in #1201
